packages/client/src/admin/Group/GroupList.tsx

```diff
@@ -71,28 +77,58 @@ export function GroupList(props: Props) {
   const edit = id => {
     history.push(`group/${id}`);
   };
-  const remove = (id, record) => {
+  const remove = async (id, record, client) => {
     const { deleteGroup } = props;
-    const { name } = record;
-    confirm({
-      title: `Delete Group`,
-      content: (
-        <span>
-          Do you really want to delete group: "<b>{name}</b>"?
-        </span>
-      ),
-      iconType: 'info-circle',
-      okText: 'Yes',
-      okType: 'danger',
-      cancelText: 'No',
-      maskClosable: true,
-      onOk() {
-        return deleteGroup({ variables: { where: { id } } });
-      },
-    });
+    const variables = { where: { id } };
+    try {
+      const result = await client.query({
+        query: GroupResourcesToBeDeleted,
+        fetchPolicy: 'no-cache',
+        variables,
+      });
+
+      const resources = result.data?.groupResourcesToBeDeleted;
+      const resourceCount = Object.values(resources)
+        .filter(value => typeof value === 'number')
+        .reduce((prev, curr) => prev + curr, 0);
+      const messageForDeleted =
+        resourceCount > 0 ? (
+          <>
+            You'll permenently lose your:
+            <ul style={{ marginBottom: 0 }}>
+              {resources.jobs ? <li>{resources.jobs} Jobs</li> : <></>}
+              {resources.schedules ? <li>{resources.schedules} Recurring Jobs</li> : <></>}
+              {resources.apps ? <li>{resources.apps} Apps</li> : <></>}
+              {resources.deployments ? <li>{resources.deployments} Deployemnts</li> : <></>}
+            </ul>
+          </>
+        ) : (
+          <></>
```

Comment:
If applying the above suggestion, this one can be removed.

Reply:
Fixed